igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerMonitor.groovy

```diff
@@ -187,6 +192,27 @@ class DockerMonitor extends CommonPollingMonitor<ImageDelta, DockerPollingDelta>
             account: image.account,
           ), artifact: dockerArtifact))
         }
+
+        if (keelService.isPresent()) {
+          String imageReference = image.repository + ":" + image.tag
+          Artifact artifact = new Artifact(
+            "DOCKER_IMAGE",
+            false,
+            image.repository,
+            image.tag,
+            image.account,
+            imageId,
+            [fullname: imageReference, registry: image.account, tag: image.tag],
+            null,
+            image.registry,
+            null
+          )
+          Map artifactEvent = [
+            payload: [artifacts: [artifact], details: [:]],
+            eventName: "spinnaker_artifacts_docker"
+          ]
+          AuthenticatedRequest.allowAnonymous { keelService.get().sendArtifactEvent(artifactEvent) }
```

Comment:
Is the `allowAnonymous` bit common practice/OK?

Reply:
It's common for service to service calls, i.e. we're not doing this call on behalf of a user so we have no user auth info to pass on.